tests/async/test_launcher.py

```diff
@@ -102,142 +85,6 @@ async def test_browser_type_name_should_work(
         raise ValueError("Unknown browser")
 
 
-async def test_browser_is_connected_should_set_connected_state(
-    browser_type, launch_arguments
-):
-    browser_server = await browser_type.launchServer(**launch_arguments)
-    remote = await browser_type.connect(wsEndpoint=browser_server.wsEndpoint)
-    assert remote.isConnected()
-    await remote.close()
-    assert remote.isConnected() is False
-    await browser_server.close()
-
-
-async def test_browser_is_connected_should_throw_when_used_after_isConnected_returns_false(
```

Comment:
Is it possible to skip these tests for now, so that we can restore coverage later?